monitoring-engine/src/main/java/io/jenkins/plugins/remotingopentelemetry/engine/log/BatchLogProcessor.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Most of this class is just a copy of the `io.opentelemetry.sdk.logging.export.BatchLogProcessor`